src/current/advisories/a106617.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Technical Advisory 106617
+advisory: A-106617
+summary: When the encryption-at-rest registry rolls over, the append-only log file contains a bug that could result in loss of the store. 
+toc: true
+affected_versions: v21.2.x, ≤ 22.1.21, ≤ 22.2.12, ≤ 23.1.7
+advisory_date: 2023-08-11
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+A CockroachDB store with [encryption-at-rest](../v23.1/security-reference/encryption.html#encryption-at-rest) enabled maintains per-file metadata necessary for file decryption within an append-only log. When this [log file](../v23.1/configure-logs.html) grows large, the log file is rotated by writing a snapshot of the metadata to a new, replacement log file. The introduction of this log file in [v21.2.0](../releases/v21.2.html) contains a bug in log rotation that could result in the omission of a single file’s metadata from the persisted log file. When a node encounters this bug, it continues to operate normally until the process is restarted. If the omitted file still exists after a process restart, the [`cockroach` process](../v23.1/cockroach-node.html) cannot decrypt or read the file, and the local store appears to be corrupted. This bug most frequently causes the `cockroach` process on a node to exit shortly after it starts. An error such as “`pebble/table: invalid table (bad magic number: 0x08cf6caea91de887)`” is logged. 
```

Comment:
The log file referred to in "When this log file grows large," is a different kind of log file than the one described in `../v23.1/configure-logs.html`. The log files described in `../v23.1/configure-logs.html` are diagnostic, only intended for operators to examine, whereas the encryption-at-rest log file is binary and only intended for CockroachDB itself to read during start. I think we should remove the link on "log file."